backend/src/models/db/test_constants.js

```diff
@@ -0,0 +1,94 @@
+exports.TEST_DB = './test.db'
+
+exports.NUM_DUMMY_USERS = 50
+
+exports.UNIVERSITY_ID = 1
+
+exports.SAMPLE_QUESTIONS = [
```

Comment:
My only feeling on this was that these questions might be pinned for all courses, or at least a few of them.

Reply:
.. and thus maybe don't belong in test_constants. But whatevs. These in theory will eventually move to prod.

Reply:
Oh I see...